.bazelci/intellij-ue.yml

```diff
@@ -10,3 +24,54 @@ platforms:
       - --test_output=errors
     test_targets:
       - //:ijwb_ue_tests
+  IntelliJ-UE-internal-canary:
```

Comment:
Here and also for CE: intellij-ce-canary and intellij-ue-canary are comparable to oss-under-dev. Hence, we should either not include them here or also use the soft-fail strategy. (Note: android-studio-canary is a bit different. See above.)

Reply:
I updated it.